.gitignore

```diff
@@ -1,3 +1,4 @@
 result
 result-*
 
+/workspace
```

Comment:
This feels like an accidental addition.

Would using `.git/info/exclude` work for you?

 - https://git-scm.com/docs/gitignore

Reply:
Oops, good catch. I'll switch to using `.git/info/exclude` for my local ignore entries.